README.md

```diff
@@ -1,98 +1,143 @@
-myip="0.0.0.0"
-cmin=0
-
-while true; do
-
-    # Check connection and reconnect
-    check=`netstat -a | grep "adsbhub[.]org[.]5001 \|adsbhub[.]org:5001 \|data[.]adsbhub[.]org[.]5001 \|data[.]adsbhub[.]org:5001 "`
-    #check=`netstat -an | grep "94[.]130[.]23[.]233[.]5001 \|94[.]130[.]23[.]233:5001 "`
-
-    if [ ${#check} -ge 10 ]
-    then
-      result="connected"
-    else
-      result="not connected"
-      eval "${cmd}" &
-    fi
-
-    #echo $result
-
-
-    # Update IP if change
-    if [ -n "$ckey" ]
-    then
-      cmin=$((cmin-1))
-      if [ $cmin -le 0 ]
-      then
-        cmin=5
-        currentip=`timeout -s KILL 5 wget -o /dev/null --no-check-certificate -qO- https://data.adsbhub.org/getmyip.php`
-
-        if [ ${#currentip} -ge 7 ] && [ "$currentip" != "$myip" ]
-        then
-          skey=`timeout -s KILL 5 wget -o /dev/null --no-check-certificate -qO- https://www.adsbhub.org/key.php`
-          if [ ${#skey} -ge 33 ]
-          then
-            ss=${skey: -1}
-            skey=${skey::-1}
-            md5=`echo -n $ckey$skey | md5sum | awk '{print $1}'`
-
-            result=`timeout -s KILL 5 wget -o /dev/null --no-check-certificate -qO- "https://www.adsbhub.org/updateip.php?sessid=$md5$ss&myip=$currentip"`
-
-            if [ "$result" == "$md5$ss" ]
-            then
-              myip=$currentip
-              #echo $result
-            fi
-	  fi
-	fi
-      fi
-    fi
-
-    sleep 60
-    
-done
+    image: mikenye/adsb-hub:latest
+    tty: true
+    container_name: adsbhub
+    restart: always
+    environment:
+      - TZ=Australia/Perth
+      - BEASTHOST=readsb
+      - SHARECODE=zg84632abhf231
+```
+
+## Up-and-Running with Docker Compose, including `mikenye/readsb`
+
+```yaml
+version: '2.0'
+
+networks:
+  adsbnet:
+
+services:
+
+  readsb:
+    image: mikenye/readsb:latest
+    tty: true
+    container_name: readsb
+    restart: always
+    devices:
+      - /dev/bus/usb/001/007:/dev/bus/usb/001/007
+    networks:
+      - adsbnet
+    command:
+      - --dcfilter
+      - --device-type=rtlsdr
+      - --fix
+      - --forward-mlat
+      - --json-location-accuracy=2
+      - --lat=-33.33333
+      - --lon=111.11111
+      - --metric
+      - --mlat
+      - --modeac
+      - --ppm=0
+      - --net
+      - --stats-every=3600
+      - --quiet
+      - --write-json=/var/run/readsb
+
+  pfclient:
+    image: mikenye/planefinder:latest
+    tty: true
+    container_name: pfclient
+    restart: always
+    environment:
+      - TZ=Australia/Perth
+      - BEASTHOST=readsb
+      - SHARECODE=zg84632abhf231
+    networks:
+      - adsbnet
+```
+
+For an explanation of the `mikenye/readsb` image's configuration, see that image's readme.
+
+## Ports
+
+No ports are exposed in this container
+
+## Environment variables
+
+| Variable | Description | Default |
+|----------|-------------|---------|
+| `TZ` | Timezone for the container. | Unset|
+| `BEASTHOST` | Host for RAW ADSB packets. Required. | Unset |
```

Comment:
Looking at ADSBHub's "how to feed" page, it says they support data in "raw Beast hex format or SBS/30003". 

Something to consider, should we also have a `BASESTATIONHOST`/`PORT` or make the variable name more generic? `DATAHOST`/`PORT`?